src/modules/ModuleGovernance/Proposal/index.vue

```diff
@@ -0,0 +1,91 @@
+<script setup lang="ts" name="ModuleGovernanceProposal">
+import { Address } from '@/core/kaikas'
+import { useProposalQuery } from '../query.proposal'
+import { Proposal } from '../types'
+import { getProposalStatus } from '../utils'
+
+const vBem = useBemClass()
+const route = useRoute()
+
+const proposalId = computed(() => route.params.id as Address)
```

Comment:
```suggestion
const proposalId = computed(() => parseAddress(route.params.id))
```

This is the exact place where we should **parse** address.

Reply:
Done